File: app/src/main/java/app/fit/fitndflow/data/dto/UserDto.java
package app.fit.fitndflow.data.dto;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class UserDto implements Serializable {
    @SerializedName("userName")
    public String userNAme;

    @SerializedName("apiKey")
    public
    String apiKey;

    @SerializedName("email")
    public String email;

    @SerializedName("premium")
    public String premium;


}

File: app/src/main/java/app/fit/fitndflow/data/dto/UserDto.kt
package app.fit.fitndflow.data.dto

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class UserDto(@SerializedName("userName") var userName: String?, @SerializedName("email") var email: String?, @SerializedName("premium") var premium: String?, @SerializedName("apiKey") var apiKey: String?): Serializable
